tomviz/WebExportWidget.h

```diff
@@ -39,10 +42,7 @@ class WebExportWidget : public QDialog
   WebExportWidget(QWidget* parent = nullptr);
   ~WebExportWidget();
 
-  QString getOutputPath();
-  int getExportType();
-  int getNumberOfPhi();
-  int getNumberOfTheta();
+  Python::Dict getKeywordArguments();
```

Comment:
Python::Dict is a light weight object that can be passed around by value without a problem.